src/Config/RectorConfig.php

```diff
@@ -138,10 +138,14 @@ public function removeUnusedImports(bool $removeUnusedImports = true): void
         SimpleParameterProvider::setParameter(Option::REMOVE_UNUSED_IMPORTS, $removeUnusedImports);
     }
 
-    public function importNames(bool $importNames = true, bool $importDocBlockNames = true): void
-    {
+    public function importNames(
+        bool $importNames = true,
+        bool $importDocBlockNames = true,
+        bool $importInsertSorted = false,
```

Comment:
I have set this new option by default to false. I would prefer that it was set to true by default but I will let you choose. If it is set to true by default we may have to modify some existing tests to reflect this new sorting